src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/CompletionHandler.cs

```diff
@@ -119,6 +120,7 @@ internal class CompletionHandler : IRequestHandler<CompletionParams, SumType<Com
             {
                 // Set some context on the CompletionItem so the CompletionResolveHandler can handle it accordingly.
                 result = SetResolveData(result.Value, serverKind);
+                result = IncludeCSharpKeywords(result.Value, serverKind);
```

Comment:
Only want to do this if the projection was C# right now you're adding the keywords even to HTML completions